aqua-csp-on-ubuntu-vm/README.md

```diff
@@ -0,0 +1,40 @@
+# A sample to deploy Ubuntu VM , install docker-ce and deploy Aqua CSP container using CustomScript Extension
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Faqua-csp-on-ubuntu-vm%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Faqua-csp-on-ubuntu-vm%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+[CustomScript Extension](https://github.com/Azure/azure-linux-extensions/tree/master/CustomScript) allows the owner of the Azure Virtual Machines to run customized scripts in the VM.
+
+This template deployed: Ubuntu VM,Storage account,Public IP address,Network interface, VNET. 
```

Comment:
This template deploys: Ubuntu VM, Storage account, Public IP address, Network interface, VNET.